CMakeLists.txt

```diff
@@ -122,7 +141,8 @@ elseif(lower_type MATCHES "relwithdebinfo")
     set(ILLIXR_BUILD_SUFFIX ".optdbg")
 endif()
 
-set(CMAKE_PREFIX_PATH "${CMAKE_INSTALL_PREFIX}/lib/cmake ${CMAKE_INSTALL_PREFIX}/lib64/cmake")
+set(Vulkan_GLSLANG_VALIDATOR_EXECUTABLE "")
+set(CMAKE_PREFIX_PATH "${CMAKE_INSTALL_PREFIX}/lib/cmake ${CMAKE_INSTALL_PREFIX}/lib64/cmake /usr/lib/x86_64-linux-gnu/cmake /usr/lib/x86_64-linux-gnu/")
```

Comment:
It is best to not specify hard coded paths here. There is no guarantee that ever OS contains these specific paths. Also these particular ones should be included by CMake itself.

Reply:
Yes you are right. There could be some leftover code that is specific to my system when I tried to build the code. Could you please correct them to what they should to be?